cryptoki/src/functions/random.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2021 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+//! Functions used to generate random numbers
+
+use crate::get_pkcs11;
+use crate::types::function::Rv;
+use crate::types::session::Session;
+use crate::Result;
+use std::convert::TryInto;
+
+impl<'a> Session<'a> {
+    /// Generates a random number and sticks it in a slice
```

Comment:
Could you also mention here that the length of the slice is used to decide how many bytes are returned?